pkg/asset/rhcos/image.go

```diff
@@ -82,6 +82,12 @@ func osImage(config *types.InstallConfig) (string, error) {
 	case azure.Name:
 		osimage, err = rhcos.VHD(ctx)
 	case baremetal.Name:
+		// Check for RHCOS image URL override
+		if osimage = config.Platform.BareMetal.ClusterMachineOSImage; osimage != "" {
+			logrus.Info("Found an override in Baremetal Platform for Cluster Machine OS Image.")
```

Comment:
As mentioned in previous comment, we should probably include the osimage override value to make this log more useful for debugging/support.

Reply:
Ditto here.